services/user.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None) -> None:
+    new_user = get_user_model().objects.create_user(username=username,
+                                                    password=password)
+    if email:
+        new_user.email = email
+    if first_name:
+        new_user.first_name = first_name
+    if last_name:
+        new_user.last_name = last_name
+    new_user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None) -> None:
+    user = User.objects.get(id=user_id)

```

Comment:
use best practice way to get user

Reply:
which?

Reply:
![image](https://user-images.githubusercontent.com/72562232/198013357-b2815459-c26d-4c72-ae9e-b048bf9ee319.png)